uptane/clients/secondary.py

```diff
@@ -620,13 +619,15 @@ def process_partial_metadata(self, director_targets_metadata):
           'that there is a man in the middle attack or misconfiguration.')
     
     targets = metadata_file_object['signed']['targets']
+
     #Combs through the director's targets metadata to find the one assigned to the
     #current ECU. 
     for target in targets: 
       if targets[target]['custom']['ecu_serial'] == self.ecu_serial:
         target_metadata['filepath'] = target
         target_metadata['fileinfo'] = targets[target]
         validated_targets_for_this_ecu.append(target_metadata)
+
```

Comment:
I'm confused: how did these whitespace changes "stop Travis CI from working"? (Was there a conflict or something?)

Reply:
Oh, I believe GitHub covered up the changes. I added some old code I had deleted in a previous commit. The deletion was causing the issue, hence, I put the code back and it worked!

Reply:
[I'm looking at one commit](https://github.com/uptane/uptane/pull/143/commits/869f44fadf660b859cc71e1171936e9c2503e7ca), not a combination of commits. How would it be covered up?

Reply:
This is strange, I can't find the commit where I deleted lines 597 through 607 in order to have less code, since I thought that should work. On closer inspection I realized that it wouldn't and basically reverted the changes to what it was originally. 
I could rename the commit if it is an issue?
Thanks!